integrations/access/msteams/configure.go

```diff
@@ -79,76 +85,77 @@ func Configure(targetDir, appID, appSecret, tenantID string) error {
 
 	printStep(&step, "Created target directory: %s", targetDir)
 
-	if err := renderTemplateTo(confTpl, p, path.Join(targetDir, "teleport-msteams.toml")); err != nil {
+	configWriter, err := os.Create(filepath.Join(targetDir, "teleport-msteams.toml"))
+	if err != nil {
 		return trace.Wrap(err)
 	}
-	if err := renderTemplateTo(manifestTpl, p, path.Join(targetDir, "manifest.json")); err != nil {
+	defer configWriter.Close()
+	if err := renderTemplateTo(configWriter, confTpl, p); err != nil {
 		return trace.Wrap(err)
 	}
 
-	printStep(&step, "Generated configuration files")
-
-	a, err := assets.ReadDir("_tpl")
+	zipWriter, err := os.Create(filepath.Join(targetDir, "app.zip"))
```

Comment:
Nit: This is technically not "zip writer". This is just a file.
```suggestion
	appZipFile, err := os.Create(filepath.Join(targetDir, "app.zip"))
```